app/controllers/shopify_app/webhooks_controller.rb

```diff
@@ -8,7 +8,7 @@ class WebhooksController < ActionController::Base
     def receive
       params.permit!
       job_args = { shop_domain: shop_domain, webhook: webhook_params.to_h }
-      webhook_job_klass.perform_later(job_args)
```

Comment:
Perhaps it would be clearer to inline the arguments i.e.:

`webhook_job_klass.perform_later(shop_domain: shop_domain, webhook: webhook_params.to_h)`

Reply:
Yep, even better. Added to this PR.